case-studies/README.md

```diff
@@ -0,0 +1,24 @@
+# SIG-Security Case Studies
+
+## Objective
+
+SIG-Security work ought to be informed by data and feedback on what works and what doesn't.
+This will help create a feedback loop between the theoretical/hypothetical and practical/tactical.
+This falls into 2 categories: process examples (e.g. how well did an assessment go?) and security practices examples (irrespective of the assessment, how are tools working out in the wild?)
+
+## Vision
+* Document example case studies, both uses of where things went right, and where things went wrong
+* Define a process for protecting the guilty (for the wrong cases) and preventing spam (for vendors or FOSS projects just looking to market their wares indesciriminately)
+* Define a process for "admission control" for what a case study should look like to add value (not too detailed, not too hand-wavy)
```

Comment:
FYI: here's the list we've done so far 
https://github.com/cncf/sig-security/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Ausecase-presentation+

This would be a great place to include links to the completed ones.  

Maybe this could be the page that is used for the presentation index for the micro-site 
cc @mhausenblas